README.md

```diff
@@ -39,6 +39,25 @@ Secret Key: <your-collector-secretkey>
 
 Run the service, replacing the env variables below like `<archiver-ip>` with the values provided to you by your contact at Shardeum
 
+`NETWORK` can be `mainnet` `stagenet` or `testnet`, you can also put `custom` and fill in the missing envs yourself.
+```bash
+docker run -p 8080:8080 -p 9001:9001 -p 10001:10001 -p 4000:4000 -p 6100:6100 -p 4446:4446 -p 4444:4444 -p 6101:6101 -d \
+  -v shardeum_db:/home/node/shardeum/db \
+  -v relayer_collector_db:/home/node/relayer-collector/db \
+  -v logs:/home/node/.pm2 \
+  -e COLLECTOR_PUBKEY=<your-collector-pubkey> \
+  -e COLLECTOR_SECRETKEY=<your-collector-secretkey> \
+  -e RMQ_PASS=<rmq-pass> \
+  -e RMQ_CYCLES_QUEUE_NAME=<rmq-cycles-queue-name> \
+  -e RMQ_RECEIPTS_QUEUE_NAME=<rmq-receipts-queue-name> \
+  -e RMQ_ORIGINAL_TXS_QUEUE_NAME=<rmq-original-txs-queue-name> \
+  -e NETWORK=testnet
```

Comment:
why would user have to pass network? 
I thought we'll have images per network. Won't that be a better approach than passing network?

Reply:
we can do either or with this setup, since the real configuration happens when it runs.